embedded-graphics/src/primitives/line/thick_points.rs

```diff
@@ -84,11 +88,14 @@ pub(in crate::primitives::line) struct ParallelsIterator {
 
     /// The next side which will be drawn.
     next_side: Side,
+
+    /// Stroke alignment.
+    alignment: StrokeAlignment,
```

Comment:
Alignment isn't supported in this PR. I left some comments in the description about this - it'll be required for e.g. thick `Triangle`s and probably `Polygon`s which are closed shapes and, for consistency with the existing closed shapes, should really support stroke alignment.

Reply:
p.s. I also added a test that ensures thick polylines are always drawn centered, regardless of alignment.

Reply:
> Alignment isn't supported in this PR.

Not for drawing any user visible objects, but if I understand the code correctly it is already supported by `ParallelsIterator`. And that file should have unit tests for the different alignments. But because it isn't used at the moment a simple `TODO: add tests for different alignments` would also be enough IMO.